### DISCUSSION TITLE
Class to disable TailwindCSS for an element and all its children

### DISCUSSION THREAD

**Discussion by adilizm** [tailwindlabs/tailwindcss#11723] [Ideas]
i was developing my app using Vuejs and tailwind and when i use vue-select component all default styles for that select are deleted i want a class to avoid tailwind to not affect this component and its children, is there any trick to do

**Comment by mybearworld** [comment]
No, that's not a thing you can do at the moment. You can just add the styles back yourself, in a way that fits your theme.
